admin/views/pupil-pin/generate-pins-overview.ejs

```diff
@@ -70,16 +70,50 @@
         </details>
         </p>
 
-        <form action="/pupil-pin/generate-<%= pinEnv %>-pins-list" name="generate-pins-list">
-            <div class="form-group">
-                <div class="form-buttons">
-                    <% if (error) { %>
-                    <input type="submit" class="button" disabled value="Generate PINs"/>
-                    <% } else { %>
-                    <input type="submit" class="button" value="Generate PINs"/>
-                    <% } %>
+        <% if(pupils && pupils.length > 0) { %>
+          <table id="generatePins" role="listbox" aria-label="Select pupils.">
+              <thead>
+              <tr>
+                  <th scope="col">
+                    Generated PIN
+                  </th>
+              </tr>
+              </thead>
+              <tbody>
+              <tr>
+                  <td>
+                      <label>
+                          PINs have been generated for <%= pupils.length %> pupils
+                      </label>
+                      <div class="font-greyed-out font-xsmall">
+                          Expires 4pm today
```

Comment:
@danwilliams same question as the other PR.
Is this supposed to be 4pm or end of the day as description to the user?

Reply:
Again this is down to the prototypes.

Reply:
@danwilliams can you elaborate what you mean with regard to prototypes?